src/Command/BaseCommand.php

```diff
@@ -90,13 +90,15 @@ protected function execute(
             );
         }
 
-        $this->configuration = $this->configurationService->loadFromDirectory(
+        $configuration = $this->configurationService->loadFromDirectory(
             $targetDirectory
         );
-        if ($this->configuration === null) {
-            $this->configuration = new ApplicationConfiguration();
+        if ($configuration === null) {
+            $configuration = new ApplicationConfiguration();
         }
 
+        $this->configuration = $configuration;
```

Comment:
Was this due to PhpStan? Would be cool, if it could detect something like this and recommends setting class members only once.

Reply:
This is due to the fact that previously the configuration was declared as nullable|ApplicationConfig. But it can be only null after reading it from the configuration and replacing it with a default configuration. But when the configuration is nullable you have to check every access.